airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlCdcProperties.java

```diff
@@ -17,6 +17,8 @@
 import java.net.URI;
 import java.nio.file.Path;
 import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Curious, is this used somewhere? I don't see any references to these imports

Reply:
Removing - I was trying to log something to debug